src/containers/ChangeTemperatureContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from "react-redux";
+import {setTemp} from "../actions"
+import ChangeTemperature from "../components/ChangeTemperature.js"
+
+
+
+function mapDispatchToProps(dispatch){
+  return{
+    set: function(temp){
+      let action = setTemp(temp);
+      dispatch(action);
+    }
+  }
+}
+
+    const ChangeTemperatureContainer = connect(mapDispatchToProps)(ChangeTemperature);
```

Comment:
Need to remember what needs to be done here when you use connect without mapStateToProps
connect(null,mapDispatchToProps)